src/sidebars/favourites/tearout-directive.js

```diff
@@ -108,9 +108,10 @@
                                         store = window.storeService.open(window.name);
                                     }
 
-                                    dragService.overAnotherInstance((overAnotherInstance) => {
+                                    dragService.overAnotherInstance('.favourites', (overAnotherInstance) => {
                                         if (overAnotherInstance) {
                                             dragService.moveToOtherInstance(scope.stock);
+                                            dragService.destroy();
                                             store.remove(scope.stock);
                                             dragService = null;
```

Comment:
It feels weird having both a `dragService.destroy()` and a `dragService = null` like that. I feel like this API is homing in, just not quite sure what it should be...